common/src/main/java/org/dash/wallet/common/Configuration.java

```diff
@@ -138,20 +132,8 @@ public void clear() {
         }
     }
 
-    private int getBtcPrecision() {
-        final String precision = prefs.getString(PREFS_KEY_BTC_PRECISION, null);
-        if (precision != null)
-            return precision.charAt(0) - '0';
-        else
-            return PREFS_DEFAULT_BTC_PRECISION;
-    }
-
     public int getBtcShift() {
-        final String precision = prefs.getString(PREFS_KEY_BTC_PRECISION, null);
-        if (precision != null)
-            return precision.length() == 3 ? precision.charAt(2) - '0' : 0;
-        else
-            return PREFS_DEFAULT_BTC_SHIFT;
+        return PREFS_DEFAULT_BTC_SHIFT;
     }
```

Comment:
These prefs never change. If we add precision settings back we might want to put them in a separate config class.